modules/cnf-configuring-the-ptp-fast-event-publisher.adoc

```diff
@@ -32,10 +45,20 @@ spec:
     node-role.kubernetes.io/worker: ""
   ptpEventConfig:
     enableEventPublisher: true <1>
-    transportHost: amqp://<instance_name>.<namespace>.svc.cluster.local <2>
+    transportHost: <transport_host> <2>
```

Comment:
for HTTP, this property can be either dropped or left blank . I prefer to be left out

Reply:
done